index.html

```diff
@@ -586,5 +598,79 @@ <h3 class="text-[20px] font-medium text-black">
       </div>
     </section>
     <!--* END ABOUT MYSELF -->
+
+    <!--* footer -->
+    <footer
+      class="flex flex-col justify-between items-center bg-light-blue w-full overflow-hidden px-5 rounded-tl-[90px] before:content-[''] before:h-[690px] before:w-[65%] before:bg-mobile-footer-shapes before:bg-contain before:-z-1 before:absolute before:left-[35%] before:bg-right before:bg-no-repeat before:mt-5"
```

Comment:
[OPTIONAL] ⭐ Great job. It is showing the background images as per the template!! However, I strongly recommend you add [a custom class](https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes) so you keep the HTML a bit cleaner. That is a downside of using Tailwind, you might end up cluttering your HTML with a lot of classes.